Signum.React/Scripts/Lines/EntityBase.tsx

```diff
@@ -264,6 +267,42 @@ export class EntityBaseController<P extends EntityBaseProps> extends LineBaseCon
     }).done();
   };
 
+  handlePasteClick = (event: React.SyntheticEvent<any>) => {
+
+    event.preventDefault();
+
+    navigator.clipboard.readText()
+      .then(text => {
+        const lites = parseLiteList(text);
+        if (lites.length == 0)
+          return;
+
+        const ti = this.props.type!.name == IsByAll ? getTypeInfo(lites[0].EntityType) : getTypeInfos(this.props.type!).singleOrNull(ti => ti.name == lites[0].EntityType);
+        if (!ti)
+          return;
+
+        const promise = lites.length == 1 ? Promise.resolve(lites[0]) : 
+          Navigator.API.fillToStrings(...lites)
+            .then(() => SelectorModal.chooseLite(new Type<Entity>(lites[0].EntityType), lites));

```

Comment:
There are three problems here: 
* you are not calling fillToStrings for the length == 1 case
* SelectorModal.chooseLite is silent if there is only one lite, no need to split the cases.
* SelectorModal.chooseLite should take a Type<T> | TypeInfo | string as first parameter

Reply:
Excellent catches 👍 , fixed